### ISSUE TITLE
TeachingTip is not accessible by screen readers

### ISSUE THREAD

**Issue by FrayxRulez** [microsoft/microsoft-ui-xaml#7992]
### Describe the bug

TeachingTip is not accessible by screen readers, that are not able to access the content of the popup by using basic keyboard navigation.

### Steps to reproduce the bug

1. Open Windows Narrator
2. Open a TeachingTip in the sample gallery
3. Notice Narrator not reading the content of the popup

### NuGet package version

WinUI 2 - Microsoft.UI.Xaml 2.8.1

**Comment by FrayxRulez** [comment]
Still not accessible
